src/options.c

```diff
@@ -485,8 +486,8 @@ bool get_cfg(options_t *opt, int argc, char *const *argv, bool shadow_enable,
 		P_CASEBOOL(267, detect_rounded_corners);
 		P_CASEBOOL(268, detect_client_opacity);
 		case 269:
-			log_warn("--refresh-rate has been deprecated, please remove it from"
-			         "your command line options");
+			log_warn("--refresh-rate has been deprecated, please remove it "
+				 "from your command line options");
```

Comment:
Just a nitpick on the formatting change here: the hanging indent should be just spaces and no tab, to follow the (deactivated) clang-format rules. :wink:
Same in line 516 below.

Reply:
addressed. i'm not sure if it's possible, but it'd be good to avoid deactivating `clang-format` at all so everything will be formatted automatically.